### PR TITLE
MAT-6202: Change TestCase Table Subtitle

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-6202](https://jira.cms.gov/browse/MAT-6202)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataTypeCell.tsx [hunk 1]
```diff
@@ -40,7 +40,9 @@ const DataTypeCell = (props: { element: DataElement; codeSystemMap: any }) => {
         </span>
       </div>
       <div className="element-type">
-        {_.startCase(element?._type.split("QDM::")[1])}
+        {element?.description
+          ? element.description.substring(element.qdmTitle.length + 2)
```
Comment (ethankaplan) [inline src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataTypeCell.tsx]: Part of a condition in case it's null, can change if unneeded

Comment (adongare) [reply src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataTypeCell.tsx]: you probably don't need else part of the condition`(_.startCase(element?._type.split("QDM::")[1])})`. it doesn't add any value by displaying data type again. I would just display element name if present as:

`{element?.description && element.description.substring(element.qdmTitle.`length + 2)}

**Review by adongare — APPROVED** [review]
LGTM
